acceptance/tests/apply/augeas/hosts.rb

```diff
@@ -0,0 +1,62 @@
+test_name "Augeas hosts file" do
+
+tag 'risk:medium'
+
+  confine :except, :platform => 'windows'
+
+  step "Backup the hosts file" do
+    on hosts, 'cp /etc/hosts /tmp/hosts.bak'
+  end
+
+  # We have a begin/ensure block here to clean up the hosts file in case
+  # of test failure.
+  begin
+
+    step "Create an entry in the hosts file" do
+      manifest = <<EOF
+augeas { 'add_hosts_entry':
+  context => '/files/etc/hosts',
```

Comment:
Due to the PA-71 issue of augeas not autoloading stock lenses, the {{incl}} and {{lens}} values need to be specified in order to get AIX to pass. We should note that these lines should be removed after that bug is squashed.

```
incl => '/etc/hosts',
lens => 'Hosts.lns',

```